### PR TITLE
[cosmwasm] wormhole stub 

### PR DESCRIPTION
**Pull request by 0xfirefist**

### CODE REVIEW

File: target_chains/cosmwasm/wormhole-stub/Dockerfile.wormhole [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Run with:
+# docker build -f Dockerfile.wormhole -o artifacts .
+
+FROM cosmwasm/workspace-optimizer-arm64:0.12.11@sha256:a00c01c2fdae8c74d6a9db95fa7b0cbb8e81101ce23a78487012e363d632fee9 AS builder
+# FROM cosmwasm/workspace-optimizer:0.12.11@sha256:b6b0b1d5ed76dafd0ec0d3812a1e7a275316cf862fd6d0a918f14d71c4cb187f AS builder
```
Comment (jayantk) [inline target_chains/cosmwasm/wormhole-stub/Dockerfile.wormhole:5]: this seems a little janky -- do you have to comment / uncomment this depending on the architecture? (you may be able to fix this by passing the platform flag to docker, or asking users to change their shell to use emulation)

**Review by jayantk — APPROVED** [review]
Can you add documentation to the README explaining what this does, when you will need it, and how to run it?

File: target_chains/cosmwasm/wormhole-stub/Dockerfile.wormhole (hunk 1 above)
Comment (0xfirefist) [reply target_chains/cosmwasm/wormhole-stub/Dockerfile.wormhole:5]: Yes, this is the case. 
I don't think passing platform flag will work. As both of these are separate images and not one image supporting multiple architecture.

Comment (0xfirefist) [reply target_chains/cosmwasm/wormhole-stub/Dockerfile.wormhole:5]: I have added comment for it. 
Arm64 ones are only to be used while testing and not for production usage

**Pull request merged**